README.md

```diff
@@ -60,14 +52,101 @@ Spinal cord MRI data:
 - T2star tra
 Brain MRI data:
 - DWI (A-P and P-A phase encoding)
-- MPM (multi-parameter mapping)
+- T1w
 
 ## naming convention
 
 sub-<site><pathology>XXX
 
 example:
-sub-zurichDCM001
+sub-montrealDCM001
+
+<ADDITIONAL NOTES IF AVAILABLE>Note: the label `bp-cspine` is used to differentiate spine images from brain.
+
+## derivatives
+
+The derivatives will be organized according to the following:
+
+https://github.com/ivadomed/ivadomed/wiki/repositories#derivatives
+
+Convention for derivatives JSON metadata:
+
+{
+  "Author": "Firstname Lastname",
+  "Date": "YEAR-MM-DD HH:MM:SS"
+}
+
+## <IF DATA ARE MISSING FOR SOME SUBJECT(S)>missing data
+
+```
+
+</details>
+
+## `dataset_description.json`
+
+The `dataset_description.json` is a JSON file describing the dataset.
+
+<details><summary>dataset_description.json template:</summary>
+
+```json
+{
+    "BIDSVersion": "BIDS 1.8.0",
+    "Name": "<dataset_name>"
+}
+```
+
+</details>
+
+## `participants.tsv`
+
+The `participants.tsv` is a TSV file and should include the following columns:
+
+| participant_id | pathology | data_id | institution_id | institution |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| sub-001 | HC | 001 | 01 | montreal |
+
+- `participant_id` - unique participant ID
+- `pathology` - pathology name; can take values listed in the [pathology column](https://docs.google.com/spreadsheets/d/1yjcA8Z0COn4OZxusIDHjStH2DpeXvscsj-aWE2X-_sg/edit?usp=sharing)
```

Comment:
I'm unfamiliar with this part of our work. Do _most_ of our scans come with pathologies? If there's no pathology, do we leave it blank? Or write "n/a" (`bids-validator` prefers "n/a")?